src/screens/NewTodoScreen.jsx

```diff
@@ -0,0 +1,66 @@
+import React, {useState} from 'react';
+import {ActivityIndicator, Button, StyleSheet, Text, TextInput, View} from 'react-native';
+import {addTodo} from "../network/api";
+
+const NewTodoScreen = ({ route , navigation }) => {
+
+    const [todoText, setTodoText] = useState('')
+    const [isSaving, setIsSaving] = useState(false)
+
+    const saveTodo = () => {
+        setIsSaving(true)
+        addTodo(2, todoText)
+            .then(data => {
+                alert('Your ToDo has been added successfully. ID:'+data.id)
+                navigation.goBack()
+            })
+            .catch(e => {
+                alert(e.toString())
+            })
+            .finally(() => {
+                setIsSaving(false)
+            })
+    }
+
+    return (
+        <View style={styles.container}>
+            <Text style={{fontSize: 24}}>Add a new ToDo</Text>
+            <TextInput
+                multiline={true}
+                style={styles.textArea}
+                placeholder={"Write your ToDo here"}
+                onChangeText={text => {setTodoText(text)}}
```

Comment:
To avoid creating useless arrow functions on each UI update
```suggestion
                onChangeText={setTodoText}
```